examples/optimized_radial_basis_functions.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
Might this be a good notebook to include in the [examples page](https://cosmo-epfl.github.io/librascal/examples/examples.html)? If so, you can link the file into the `docs/source/examples/` directory and then add it here: https://github.com/cosmo-epfl/librascal/blob/4e576ae7b9d3740715ab1910def5e1a15ffd1268/docs/source/examples/examples.rst#L12
Then try compiling the docs locally and see if it produces any errors :)

Reply:
Works! Will push with next commit

Reply:
@ceriottm does it really work? If I compile it locally I don't see the radial optimizations notebook. I find that weird because I could not find `zundel_i-PI.ipynb` added anywhere else than here, so I am not sure why this is